core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -0,0 +1,93 @@
+package org.arquillian.smart.testing;
+
+import java.util.logging.Level;
+
+import static java.lang.String.format;
+
+public class Logger {
+
+    private java.util.logging.Logger jul;
+
+    private static String prefix = "%s: Smart-Testing - ";
+
+    private Logger(java.util.logging.Logger logger) {
+        this.jul = logger;
+    }
+
+    public static Logger getLogger(Class clazz) {
+        return getLogger(clazz.getName());
+    }
+
+    public static Logger getLogger(String name) {
+        return new Logger(java.util.logging.Logger.getLogger(name));
+    }
+
+    /**
+     * Log a message using java.util.logging.Logger
+     *
+     * @param   level
+     *     One of the message level identifiers, e.g., SEVERE
+     * @param   msg
+     *     The string message (or a key in the message catalog)
+     * @param   args
+     *     arguments to the message
+     */
+    public void log(Level level, String msg, Object... args) {
+        if (jul.isLoggable(level)) {
+            jul.log(level, msg, args);
+        }
+    }
+
+    /**
+     * Will format the given message with the given arguments and prints it on standard output with the prefix:
+     * "INFO: Smart-Testing - "
+     *
+     * @param msg
+     *     Message to print
+     * @param args
+     *     arguments to use for formatting the given message
+     */
+    public void info(String msg, Object... args) {
+        System.out.println(getFormattedMsg("INFO", msg, args));
+    }
+
+    /**
+     * Will format the given message with the given arguments and prints it on error output with the prefix:
+     * "WARN: Smart-Testing - "
+     *
+     * @param msg
+     *     Message to print
+     * @param args
+     *     arguments to use for formatting the given message
+     */
+    public void warn(String msg, Object... args) {
+        System.err.println(getFormattedMsg("WARN", msg, args));
+    }
+
+    public void severe(String msg) {
```

Comment:
add javadoc for this method and for all following...  please add also the possibility of specifying args to be used for formatting of the given message. In the same way, as it is done in case of `warn` and `info`